src/metrics.js

```diff
@@ -0,0 +1,81 @@
+const octokit = require('@octokit/rest')() 
+const fs = require('fs');
+ 
+var gitToken = "thisToken"
+ 
+octokit.repos.getForOrg({
+  org: 'octokit',
+  type: 'public'
+})
+ 
+var targetProjectRoot = "C:/Users/SHAWN/data/"
```

Comment:
This can't be hard coded, once again I'd suggest an environment variable.  That way each dev, or the server, can use whatever they want.  `process.env` gives you access to these.